episodes/05-raster-calculations.md

```diff
@@ -49,7 +49,7 @@ For this episode, we will use one of the Sentinel-2 scenes that we have already
 ## Introduce the Data
 
 We'll continue from the results of the satellite image search that we have carried out in an exercise from
-[a previous episode]({{site.baseurl}}/05-access-data). We will load data starting from the `search.json` file.
+[a previous episode](05-access-data.md). We will load data starting from the `search.json` file.
```

Comment:
I changed it in #84. but I think we can still leave it in. just mention in case we get a conflict.